### DISCUSSION TITLE
Is there a way to configure Asset Modules with esModule=false?  [webpack 5]

### DISCUSSION THREAD

**Discussion by manasb-uoe** [webpack/webpack#13323] [General]
Asking because we're migrating from file/url loaders to Asset Modules and we want to continue using esModule=false.

[a comment by an automated account is not shown]

**Reply by alexander-akait** [reply]
No, why you need this, we use different export based on `require`/`import`

**Reply by manasb-uoe** [reply]
There's a lot of imports across multiple repos which do `import * as bla from 'bla.jpg'` and then `<img src={bla} />`. These only work with file-loader when `esModule=false`. We want to keep people's current source unchanged while we do webpack 4 -> 5 upgrade at a platform level.

**Reply by alexander-akait** [reply]
You can write custom loader in this case for compatibility purpose. When you use `import * as bla from 'bla.jpg'` you will have `default` always by ES modules. Also you can continue to use `file-loader`/`url-loader`, they are deprecated, but will work.

**Reply by manasb-uoe** [reply]
Good idea, thank you!
